.devcontainer/images/Dockerfile

```
@@ -1,192 +1,155 @@
+# Dynamic Binary Tools (grepai, rtk) — kubectl/helm/bazelisk/yq are in base
 # =============================================================================
-ARG KUBECTL_VERSION=1.35.0
-ARG HELM_VERSION=4.0.4
-
-RUN ARCH_K8S=$([ "$TARGETARCH" = "arm64" ] && echo "arm64" || echo "amd64") && \
-    curl -fsSL "https://dl.k8s.io/release/v${KUBECTL_VERSION}/bin/linux/${ARCH_K8S}/kubectl" -o /usr/local/bin/kubectl && \
-    chmod +x /usr/local/bin/kubectl && \
-    curl -fsSL "https://get.helm.sh/helm-v${HELM_VERSION}-linux-${ARCH_K8S}.tar.gz" | tar xz -C /tmp && \
-    mv /tmp/linux-${ARCH_K8S}/helm /usr/local/bin/helm && \
-    rm -rf /tmp/linux-${ARCH_K8S}
+RUN ARCH=$([ "$TARGETARCH" = "arm64" ] && echo "arm64" || echo "amd64") && \
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)